# Relax Python version constraint to allow Python 3.13
# Original constraint was ">=3.12,<3.13"
# Author: Emik Lin (HKUMed CPOS)
--- pyproject.toml.orig	2025-05-07 10:46:47.000000000 +0800
+++ pyproject.toml	2026-01-23 11:48:00.547436697 +0800
@@ -12,7 +12,7 @@
 ]
 license = {text = "BSD 3-clause"}
 readme = "README.md"
-requires-python = ">=3.12,<3.13"
+requires-python = ">=3.12,<3.14"
 
 dependencies = [
     "pysam (>=0.23.0,<0.24.0)",
